media-sound/nulloy/nulloy-0.9.9.ebuild

```diff
@@ -8,9 +8,10 @@ inherit xdg
 DESCRIPTION="Music player with a waveform progress bar (sound visualization)"
 HOMEPAGE="https://nulloy.com"
 
-NAME="nulloy-theme-night-v1.0"
+THEME_VERSION=1.1.1
+NAME="nulloy-theme-night-${THEME_VERSION}"
 SRC_URI="https://github.com/nulloy/nulloy/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
-	skins? ( https://gitlab.com/vitaly-zdanevich/nulloy-theme-night/-/archive/v1.0/${NAME}.tar.bz2 )"
+	skins? ( https://gitlab.com/vitaly-zdanevich/nulloy-theme-night/-/archive/${THEME_VERSION}/${NAME}.tar.bz2 )"
```

Comment:
I think will make more sense to make a PR to include the theme

Reply:
Hm... lets merge this pull request for now please...

Reply:
I am not a fan either of mixing sources here; we should let users choose if they want the upstream nulloy release separate from if they want your additional theme.